stdlib/LinearAlgebra/test/triangular.jl

```diff
@@ -329,6 +331,21 @@ for elty1 in (Float32, Float64, BigFloat, ComplexF32, ComplexF64, Complex{BigFlo
                 @test_throws DimensionMismatch transpose(A2) * offsizeA
                 @test_throws DimensionMismatch A2'  * offsizeA
                 @test_throws DimensionMismatch A2   * offsizeA
+                if( uplo1 == uplo2 && elty1 == elty2 != Int && t1 != UnitLowerTriangular && t1 != UnitUpperTriangular )
+                    @test rdiv!(copy(A1), copy(A2)) ≈ A1/A2
```

Comment:
I'm not sure, but don't you want to turn the rhs into dense matrices? `A1/A2` uses `rdiv!`, so you may end up testing the same function on both sides. But it's very hard to tell which methods are used eventually.